### PR TITLE
Wip development

### PR DESCRIPTION
**Pull request by markedwardmurray**
@misterfifths @jmburges 

New walkthrough to follow Using Tests. This is a companion to objc-palindrome-detector but using Specta instead of NSLog()ing.

Please review by Tue 9/22

regarding https://github.com/learn-co-curriculum/mobile-development-with-ios-track/issues/23

### CODE REVIEW

**Comment by AnnJohn** [comment]
Reassigned to @misterfifths since Joe is teaching this week

**Comment by markedwardmurray** [comment]
Reassigning to Joe since Tim is preparing for the new class.

File: README.md [hunk 1]
```diff
@@ -2,5 +2,87 @@
...
+![](https://curriculum-content.s3.amazonaws.com/ios/ios-objc-fundamentals-unit/palindrome_no_method_errors.png)
+
+Don't panic! This is because the test file `FISAppDelegateSpec.m` is looking for the methods that you're going to write. Objective-C doesn't have a `NoMethodError` like some other languages such as Ruby, so instead the pre-compiler simply keeps the build from succeeding in order to avoid a definite crash. The fix for this is simple: write the methods the test file is instructed to call!
+
+2 — Navigate to the `FISAppDelegate.h` header file and declare the same two methods from the previous lab:
+
+  * `stringIsPalindrome:` which takes one `NSString` argument called `string` and returns a `BOOL`, and
+  * `stringByReversingString:` which takes on `NSString` argument called `string` and returns an `NSString`.
+
+3 — Now Navigate to the `FISAppDelegate.m` implementation file and use autocomplete to define the implementation of `stringIsPalindrome:` to `return NO;` and the implementation of `stringByReversingString:` to `return nil;`.
```
Comment (jmburges) [inline README.md]: lower case Navigate

File: README.md [hunk 2]
```diff
@@ -2,5 +2,87 @@
...
+
+You can (temporarily) drag the Navigator area's pane out to the right in order to view the descriptions of each test, which can sometimes be rather long. Unfortunately there is not a way to get Xcode to wrap this text to a new line in the Navigator area so dragging the pane wider is common practice.
+
+Alternatively, you can view the test results in the Results navigator by clicking on the right-most symbol in the top of the Navigator area:
+
+![](https://curriculum-content.s3.amazonaws.com/ios/ios-objc-fundamentals-unit/palindrome_results_navigator.png) 
+
+**Note:** *The Results navigator is useful for comparing long test results that do not fit into the highlighted flags that appear in the Code Editor when viewing a test file.*
+
+5 — Insert the provided implementation for the `stringByReversingString:` method:
```
Comment (jmburges) [inline README.md]: Explain what this method does.

Comment (jmburges) [inline README.md]: like. This reverses the string!

**Comment by jmburges** [comment]
some nitpicks and :shipit:

**Comment by markedwardmurray** [comment]
:+1:

**Pull request merged**